Source/WebSocket.swift

```diff
@@ -355,6 +355,9 @@ open class WebSocket : NSObject, StreamDelegate {
                         return
                     }
                 }
+            } else {
```

Comment:
OK, how about I only submit the docs for now, and I'll try to think of how to introduce the error message in a separate later pull request?